web/package.json

```diff
@@ -6,7 +6,6 @@
   "scripts": {
     "audit": "npx audit-ci --config ./audit-ci.json",
     "make-state-icons": "babel-node scripts/stateIcons.js && cp src/data/states-small/* src/static/img/states/",
-    "copy-schemas": "cp -r ../schemas src/static",
```

Comment:
Does Tif want to get rid of this?

Reply:
we will when I get done with this workspace stuff

Reply:
Should I leave as is (deleted in my branch) or bring it back?